src/services/Content/ContentApi.ts

```diff
@@ -330,12 +330,30 @@ export const introspectRepository: (
   return data;
 };
 
+export const triggerSnapshot: (
+  repositoryUUID: string,
+  resetCount: number,
```

Comment:
you removed the one reference to resetCount, but you need to remove this one too